### PR TITLE
Fixed #5286 - Empty help message when creating user

### LINKED ISSUE #5286: Empty help message when creating user
#### Issue
When creating a user and clicking the exclamation mark next to "Module Menu Filters" an empty help message is displayed:
emptyhelp

#### Expected Behavior
Provide the user with a description of what this function does.

#### Actual Behavior
Shows an empty help box.

#### Steps to Reproduce
1. Click create user
2. Click Layout Options
3. Click the exclamation mark next to the text "Module Menu Filters"

#### Your Environment

* SuiteCRM Version used: 7.10

### PR DESCRIPTION
**Pull request by ChangezKhan**
## Description

Fixed #5286  - Empty help message when creating user

## Motivation and Context

When creating a user and clicking the exclamation mark next to "Module Menu Filters" an empty help message was displayed.

## How To Test This

1. Click create user
2. Click Layout Options
3. Click the exclamation mark next to the text "Module Menu Filters"

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: modules/Users/language/en_us.lang.php [hunk 1]
```diff
@@ -214,6 +214,7 @@
     'LBL_MODULE_TITLE' => 'Users: Home',
     'LBL_NAME' => 'Full Name',
     'LBL_SIGNATURE_NAME' => 'Name',
+    'LBL_NAVIGATION_PARADIGM_DESCRIPTION' => 'Select to be able to view modules in the navigation bar based on pre-defined groups. When selected, the "Filter Menu By" feature will appear in the "More" menu.',
```
Comment (horus68) [inline modules/Users/language/en_us.lang.php]: "Filter Menu By" feature? 
So there must be a language string with the value "Filter Menu By" but I can't find it!

Comment (ChangezKhan) [reply modules/Users/language/en_us.lang.php]: Actually, I got this string from "SuiteCRM 7.8.20".

Comment (horus68) [reply modules/Users/language/en_us.lang.php]: True... maybe the other one was also deleted. @Dillon-Brown to the rescue!

Comment (Dillon-Brown) [reply modules/Users/language/en_us.lang.php]: @horus68 Looks like this line was removed accidentally in 7.9.3. Doesn't look like there are any other missing strings related to "Filter Menu By", the string is "Configure Module Menu Filters".

**Comment by Dillon-Brown** [comment]
Assessed

**Pull request merged**